### PR TITLE
4616: Get canned data

### PR DESCRIPTION
**Pull request by RuthBerry96**
Story: [AB#4616](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/4616)
Task: [AB#7168](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/7168)

### CODE REVIEW

File: src/NHSD.BuyingCatalogue.Ordering.Api/Models/AddressModel.cs [hunk 1]
```diff
@@ -0,0 +1,15 @@
+﻿namespace NHSD.BuyingCatalogue.Ordering.Api.Models
+{
+    public sealed class AddressModel
```
Comment (bjss-chris) [inline src/NHSD.BuyingCatalogue.Ordering.Api/Models/AddressModel.cs:3]: Class can be `internal`.

File: src/NHSD.BuyingCatalogue.Ordering.Api/Models/OrderingPartyModel.cs [hunk 2]
```diff
@@ -0,0 +1,8 @@
+﻿namespace NHSD.BuyingCatalogue.Ordering.Api.Models
+{
+    public sealed class OrderingPartyModel
```
Comment (bjss-chris) [inline src/NHSD.BuyingCatalogue.Ordering.Api/Models/OrderingPartyModel.cs:3]: Class can be `internal`.

File: src/NHSD.BuyingCatalogue.Ordering.Api/Models/OrganisationModel.cs [hunk 3]
```diff
@@ -0,0 +1,9 @@
+﻿namespace NHSD.BuyingCatalogue.Ordering.Api.Models
+{
+    public sealed class OrganisationModel
```
Comment (bjss-chris) [inline src/NHSD.BuyingCatalogue.Ordering.Api/Models/OrganisationModel.cs:3]: Class can be `internal`.

File: src/NHSD.BuyingCatalogue.Ordering.Api/Models/PrimaryContactModel.cs [hunk 4]
```diff
@@ -0,0 +1,10 @@
+﻿namespace NHSD.BuyingCatalogue.Ordering.Api.Models
+{
+    public sealed class PrimaryContactModel
```
Comment (bjss-chris) [inline src/NHSD.BuyingCatalogue.Ordering.Api/Models/PrimaryContactModel.cs:3]: Class can be `internal`.

Comment (RuthBerry96) [reply src/NHSD.BuyingCatalogue.Ordering.Api/Models/PrimaryContactModel.cs]: 👍

File: src/NHSD.BuyingCatalogue.Ordering.Api/Models/OrganisationModel.cs (hunk 3 above)
Comment (RuthBerry96) [reply src/NHSD.BuyingCatalogue.Ordering.Api/Models/OrganisationModel.cs]: 👍

File: src/NHSD.BuyingCatalogue.Ordering.Api/Models/OrderingPartyModel.cs (hunk 2 above)
Comment (RuthBerry96) [reply src/NHSD.BuyingCatalogue.Ordering.Api/Models/OrderingPartyModel.cs]: 👍

File: src/NHSD.BuyingCatalogue.Ordering.Api/Models/AddressModel.cs (hunk 1 above)
Comment (RuthBerry96) [reply src/NHSD.BuyingCatalogue.Ordering.Api/Models/AddressModel.cs]: 👍

**Pull request merged**